tasks/collect-mrrc-params/collect-mrrc-params.yaml

```diff
@@ -3,6 +3,11 @@ apiVersion: tekton.dev/v1
 kind: Task
 metadata:
   name: collect-mrrc-params
+  labels:
+    app.kubernetes.io/version: "3.1.0"
```

Comment:
We typically start with version 0.1.0, so I would suggest that. 1.0.0 would be acceptable as well.

Reply:
Fixed.